### ISSUE TITLE
[Usage]: how can I use vllm to generate logits instead of text

### ISSUE THREAD

**Issue by lihongxiacream** [vllm-project/vllm#7981]
### Your current environment

```text
The output of `python collect_env.py`
```

### How would you like to use vllm

I can not find the file path where the logits generate. 

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.
